src/core/server/http/http_server.ts

```diff
@@ -221,6 +223,7 @@ export class HttpServer {
     this.authRegistered = true;
 
     const sessionStorageFactory = await createCookieSessionStorageFactory<T>(
+      this.logger.get('http', 'server', this.name, 'cookie-session-storage'),
```

Comment:
It'd be kinda nice if the `Logger` interface had a `get` function for creating sub-loggers:

```js
const httpLog = loggerFactory.get('http', 'server')
const cookieSessionStorageLog = httpLog.get('cookie-session-storage')
```

But that's a change for another day...

Reply:
I also was surprised that `Logger` doesn't provide this functionality. Created issue https://github.com/elastic/kibana/issues/39695